src/compiler.ts

```diff
@@ -884,6 +884,13 @@ export class Compiler extends DiagnosticEmitter {
                   }
                 }
               }
+              let hasVectorValueOperands = signature.hasVectorValueOperands;
+              if (hasVectorValueOperands) {
+                this.warning(
+                  DiagnosticCode.External_0_type_does_not_support_for_all_host_runtimes,
```

Comment:
Perhaps

> Exchange of 'v128' values is not supported by all embeddings.

And indicate the respective parameter node instead of the function identifier?

Reply:
I like `Exchange of 'v128' values is not supported by all embeddings.`. According "indicate the respective parameter node instead of the function identifier" it little bit harder due to need to scan parameter's position. Hmm

Reply:
Seems that'd ideally live on the function instance, obtaining all potentially unsupported parameter nodes?